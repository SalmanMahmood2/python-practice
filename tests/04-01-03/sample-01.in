add
Model A
print
remove
S14
remove
240z
print
