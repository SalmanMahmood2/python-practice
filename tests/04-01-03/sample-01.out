S14
240z
R33
350z
Model A
R33
350z
Model A